alibi_detect/tests/test_dep_management.py

```diff
@@ -136,9 +136,9 @@ def test_od_backend_dependencies(opt_dep):
     """
     dependency_map = defaultdict(lambda: ['default'])
     for dependency, relations in [
-            ('Ensembler', ['torch', 'keops']),
-            ('KNNTorch', ['torch', 'keops']),
-            ('to_numpy', ['torch', 'keops']),
+                ('Ensembler', ['torch', 'keops']),
+                ('KNNTorch', ['torch', 'keops']),
+                ('MahalanobisTorch', ['torch', 'keops'])
```

Comment:
Because it's attached as a method on the `TorchOutlierDetector` class which is inherited by `KNNTorch` and `MahalanobisTorch` so we don't need to protect it here.